pkg/keys.go

```diff
@@ -58,26 +57,32 @@ func CreateSshKeyPair() {
 
 	}
 	publicKey = viper.GetString("botpublickey")
-	privateKey := viper.GetString("botprivatekey")
-
-	var argocdInitValuesYaml = []byte(fmt.Sprintf(`
-configs:
-  repositories:
-    soft-serve-gitops:
-      url: ssh://soft-serve.soft-serve.svc.cluster.local:22/gitops
-      insecure: 'true'
-      type: gitClient
-      name: soft-serve-gitops
-  credentialTemplates:
-    ssh-creds:
-      url: ssh://soft-serve.soft-serve.svc.cluster.local:22
-      sshPrivateKey: |
-        %s
-`, strings.ReplaceAll(privateKey, "\n", "\n        ")))
-
-	err := os.WriteFile(fmt.Sprintf("%s/argocd-init-values.yaml", config.K1FolderPath), argocdInitValuesYaml, 0644)
-	if err != nil {
-		log.Panicf("error: could not write argocd-init-values.yaml %s", err)
+
+	// todo: break it into smaller function
+	if viper.GetString("gitprovider") != CloudK3d {
+
+		config := configs.ReadConfig()
+		privateKey := viper.GetString("botprivatekey")
+
+		var argocdInitValuesYaml = []byte(fmt.Sprintf(`
```

Comment:
if I am not wrong, this alignment will broke, we need to keep aligned all to left